package.json

```diff
@@ -2,7 +2,7 @@
   "name": "bridge-in-tech-web",
   "version": "0.1.0",
   "private": true,
-  "homepage": "http://localhost:3000/",
+  "homepage": "https://anitab-org.github.io/bridge-in-tech-web/",
```

Comment:
Probably, we can do something like this
```suggestion
  "homepage": "https://anitab-org.github.io/bridge-in-tech-web/", // this is the comment
```

Reply:
No, it's not possible to add comments like this it will throw an error.
![Screenshot from 2021-04-24 06-30-28](https://user-images.githubusercontent.com/58786336/115942202-aabe2500-a4c6-11eb-9d8f-13b6e7ee3460.png)

Reply:
hey, can you suggest the alternative for this @mtreacy002 .

Reply:
@shades-7 The only alternative is to add a comments property as you suggested above. Let's wait for Maya's input on this.

Reply:
@shades-7 and @jalajcodes , we can try your approach then which is to use a comment's property. Hopefully this will work 🙏.

Reply:
@mtreacy002 @jalajcodes done